jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -175,6 +175,15 @@ private static void completeInitialization() {
 		if (openj9.internal.criu.InternalCRIUSupport.isCRaCSupportEnabled()) {
 			// export java.base/jdk.crac unconditionally
 			J9VMInternals.class.getModule().implAddExports("jdk.crac"); //$NON-NLS-1$
+			try {
+				// java.base can't require any other modules
+				// use reflection to get module jdk.management export jdk.crac.management unconditionally
+				// assuming module jdk.management to be loaded when CRaC is enabled
+				Class<?> clz = Class.forName("openj9.lang.management.InvalidOptionException"); //$NON-NLS-1$
```

Comment:
I think you can use `ModuleFinder::ofSystem` to find specific modules

Reply:
`ModuleFinder::ofSystem` returns `ModuleFinder`, `ModuleFinder.find(String name)` returns `ModuleReference`, `ModuleDescriptor.descriptor()` returns `ModuleDescriptor`, I didn't find a method returning a `Module` instance, any suggestion?

Reply:
okay, perhaps something like `ModuleLayer.boot().findModule(...)`

Reply:
Use `ModuleLayer.boot().findModule("jdk.management")` to find the module in question.